docs/example.md

```diff
@@ -262,3 +262,19 @@ $ arrai eval '{|a,b| (1,2), (1,3)} <&> {|a,c| (1,2), (1,3)}'
 $ arrai eval '{|a,b,c| (1,2,2), (1,2,3)} <&> {|b,c,d| (2,3,4), (1,3,4)}'
 {(a: 1, b: 2, c: 3, d: 4)}
 ```
+
+### Merging operator
+
+Merges two tuples and combines all rhs(right-hand side) attributes and any lhs(left-hand side) attributes whose name isn't on the rhs into a new tuple.
```

Comment:
Small nit: good description but a bit of a mouthful. Two sentences is clearer, saying a) what it does, and b) addressing edge cases.

```suggestion
Merge combines two tuples, producing a single tuple containing a union of their attributes. If the same name is present in both the LHS (left-hand side) and RHS (right-hand side) tuples, the RHS value takes precedence in the output.
```